apps/src/eval/eval.js

```diff
@@ -397,12 +408,24 @@ Eval.execute = function() {
     result: Eval.result,
     testResult: Eval.testResults,
     program: encodeURIComponent(textBlocks),
-    onComplete: onReportComplete
+    onComplete: onReportComplete,
+    image: Eval.encodedFeedbackImage
   };
 
-  studioApp.playAudio(Eval.result ? 'win' : 'failure');
+  if (typeof document.getElementById('svgEval').toDataURL === 'undefined') { // don't try it if function is not defined
```

Comment:
Any chance this would get hit outside of the test environment? If not, should that comment mention this case being test-specific?